From 1084b0255c42fc86adb0a2c7614e341ac530db65 Mon Sep 17 00:00:00 2001
From: "Arnout Vandecappelle (Essensium/Mind)" <arnout@mind.be>
Date: Sun, 8 Oct 2017 00:53:31 +0200
Subject: [PATCH] libparted/labels: link with libiconv if needed

gpt.c uses iconv so it should link with it. Otherwise, on platforms
where libiconv is a separate library, we get a link failure of parted:

    CCLD     parted
  ../libparted/.libs/libparted.so: undefined reference to `libiconv'
  ../libparted/.libs/libparted.so: undefined reference to `libiconv_open'
  ../libparted/.libs/libparted.so: undefined reference to `libiconv_close'

Signed-off-by: Arnout Vandecappelle (Essensium/Mind) <arnout@mind.be>
---
 libparted/labels/Makefile.am | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/libparted/labels/Makefile.am b/libparted/labels/Makefile.am
index c996f81..b204379 100644
--- a/libparted/labels/Makefile.am
+++ b/libparted/labels/Makefile.am
@@ -35,7 +35,7 @@ liblabels_la_SOURCES = \
   rdb.c		\
   sun.c
 
-liblabels_la_LIBADD = $(OS_LIBS) $(INTLLIBS)
+liblabels_la_LIBADD = $(OS_LIBS) $(INTLLIBS) $(LIBICONV)
 
 AM_CPPFLAGS = $(partedincludedir) $(INTLINCS)
 
-- 
2.14.2

